src/app/starredentries/starredentries.component.html

```diff
@@ -85,41 +87,39 @@
         <mat-tab>
           <ng-template mat-tab-label>
             <div fxLayoutGap="0.5rem">
-              <img class="notebook-icon" src="../../../assets/svg/sub-nav/notebooks.svg" alt="notebook logo" />
+              <img class="site-icons-tab" src="../../../assets/svg/sub-nav/notebook.svg" alt="notebook logo" />
               <span>Notebooks</span>
               <span class="tab-display" data-cy="starred-notebooks-count">{{ starredNotebooks?.length }}</span>
             </div>
           </ng-template>
           <div>
-            <mat-card class="alert alert-info starred-header" role="alert" *ngIf="starredNotebooks?.length === 0">
-              <mat-icon>warning</mat-icon>
+            <mat-card
+              class="alert alert-info mat-elevation-z starred-header"
+              data-cy="no-notebooks-banner"
+              role="alert"
+              *ngIf="starredNotebooks?.length === 0"
+            >
               You have no starred notebooks.
             </mat-card>
             <div fxLayout="column" fxLayoutGap="1rem" class="mb-2 mx-1">
               <mat-card fxFlex *ngFor="let notebook of starredNotebooks; let last = last" class="m-1 pb-3">
                 <ng-container
-                  *ngTemplateOutlet="workflowSummary; context: { workflow: notebook, isAppTool: false, isNotebook: true }"
+                  *ngTemplateOutlet="workflowSummary; context: { workflow: notebook, entryType: entryType.Notebook }"
                 ></ng-container>
               </mat-card>
             </div>
           </div>
         </mat-tab>
 
-        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool" let-isNotebook="isNotebook">
+        <ng-template #workflowSummary let-workflow="workflow" let-entryType="entryType">
           <div fxLayout="row" fxLayoutGap="10px" fxLayoutAlign="space-between">
-            <a [routerLink]="(isAppTool ? entryType.AppTool : entryType.BioWorkflow) | routerLink: workflow" class="no-underline">
+            <a [routerLink]="entryType | routerLink: workflow" class="no-underline">
               <mat-card-header>
                 <mat-card-title fxLayoutAlign="space-between" fxLayoutGap="1rem">
                   <img
-                    *ngIf="isNotebook"
-                    src="../../../assets/svg/sub-nav/notebooks.svg"
-                    alt="{{ isAppTool ? 'tool' : 'workflow' }} logo"
-                    class="notebook-icon"
-                  />
-                  <img
-                    *ngIf="!isNotebook"
-                    src="../../../assets/svg/sub-nav/{{ isAppTool ? 'tool' : 'workflow' }}.svg"
-                    alt="{{ isAppTool ? 'tool' : 'workflow' }} logo"
+                    class="site-icons-small entryType-icons"
+                    src="../../../assets/svg/sub-nav/{{ entryType }}.svg"
```

Comment:
I didn't try this locally, but I don't think this will work for `EntryType.AppTool` because it should use `tool.svg`, not `appTool.svg`. You'll need a conditional to check for `AppTool` and set it to `tool`